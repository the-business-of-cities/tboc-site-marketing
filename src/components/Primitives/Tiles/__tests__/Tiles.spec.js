import { TilesWrapper } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("TilesWrapper", () => {
  it("renders correctly", () => {
    expect(shallow(<TilesWrapper />)).toMatchSnapshot();
  });
});
