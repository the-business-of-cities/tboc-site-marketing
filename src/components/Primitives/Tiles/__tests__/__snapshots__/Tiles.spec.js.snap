// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TilesWrapper renders correctly 1`] = `
.c1 {
  link: text-decoration:none;
  color: currentColor;
}

general:.c0:before,
.c1:after {
  content: "";
  display: table;
}

.c1:after {
  clear: both;
}

<div
  className="c0 c1"
/>
`;
